.github/workflows/website.yml

```diff
@@ -36,7 +36,7 @@ jobs:
           cp -r app/ gh-pages/
           cp dist/*.min.js gh-pages/
           cp chemiscope_standalone.html gh-pages/
-          cp -r docs/_build/html/ gh-pages/docs/
```

Comment:
What does this change have to do with jsmol?

Reply:
not much, I was checking the docs and made a small change on the side. I'll extract the change to its own commit